x-pack/plugins/security_solution/server/usage/detections/detections_helpers.ts

```diff
@@ -22,6 +21,26 @@ interface DetectionsMetric {
   isEnabled: boolean;
 }
 
+interface RuleSearchBody {
+  query: {
+    bool: {
+      filter: {
+        term: { [key: string]: string };
+      };
+    };
+  };
+}
+interface RuleSearchParams {
```

Comment:
Nit: since `ruleSearchOptions` is an internal variable and its type is already constrained by its usage I would be inclined to make its type implicit and remove `RuleSearchParams`. However, there's nothing _wrong_ with the explicit interface here, either.

Reply:
I'll keep it explicit for now and leave it up to your team to remove if that's what you prefer.